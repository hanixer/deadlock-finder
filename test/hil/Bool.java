public class Bool {
    static void func1() {
        boolean b1 = true;
        boolean b2 = false;
        if (b1) {
            b2 = true;
        }
    }
}