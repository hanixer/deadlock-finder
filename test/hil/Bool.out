func Bool.func1(): void {
  var b1: boolean = true
  var b2: boolean = false
  if (b1) {
    b2 = true
  }
}